src/platform/node/metrics.js

```diff
@@ -33,12 +35,13 @@ module.exports = function () {
 
       client = new StatsD({
         host: this._config.hostname,
-        port: 8125, // TODO: allow to configure this
+        port: this._config.dogstatsd.port,
         prefix: 'runtime.node.',
         globalTags: {
           'env': this._config.env,
           'service': this._config.service,
-          'runtime-id': this._config.runtimeId
+          'runtime-id': this._config.runtimeId,
+          'language': 'javascript'
```

Comment:
We don't need that one for runtime metrics, at this point you can just set something such as `runtime: node`, or just wait that we spec this across languages.